core/spec/models/spree/customer_return_spec.rb

```diff
@@ -107,6 +120,19 @@
     end
   end
 
+  describe '#currency' do
+    subject { customer_return.currency }
+
+    let(:return_item) { create(:return_item) }
+    let(:customer_return) { build(:customer_return, return_items: [return_item]) }
+    let(:order) { create(:order, currency: test_currency) }
+    let(:test_currency) { 'TEST' }
```

Comment:
```suggestion
    let(:test_currency) { 'USD' }
```

let's operate on real currencies as in the future we will likely add some validation here